jdbc-perf-logger-gui/src/main/java/ch/sla/jdbcperflogger/console/ui/CustomTableCellRenderer.java

```diff
@@ -44,38 +44,61 @@ public Component getTableCellRendererComponent(@Nullable final JTable table, @Nu
-                break;
-            case NON_PREPARED_QUERY_STMT:
-                component.setForeground(Color.RED);
-                component.setText("Q");
-                break;
-            case NON_PREPARED_BATCH_EXECUTION:
-                component.setForeground(Color.MAGENTA);
-                component.setText("B");
-                break;
-            case PREPARED_BATCH_EXECUTION:
-                component.setForeground(Color.BLUE);
-                component.setText("PB");
-                break;
-            case BASE_PREPARED_STMT:
-                component.setForeground(Color.CYAN);
-                component.setText("PS");
-                break;
-            case PREPARED_QUERY_STMT:
-                component.setForeground(Color.GREEN);
-                component.setText("PQ");
-                break;
-            case TRANSACTION:
-                component.setForeground(Color.DARK_GRAY);
-                component.setText("TX");
-                break;
+                case BASE_NON_PREPARED_STMT:
+                    component.setForeground(Color.ORANGE);
+                    component.setText("S");
+                    break;
+                case NON_PREPARED_QUERY_STMT:
+                    component.setForeground(Color.RED);
+                    component.setText("Q");
+                    break;
+                case NON_PREPARED_BATCH_EXECUTION:
+                    component.setForeground(Color.MAGENTA);
+                    component.setText("B");
+                    break;
+                case PREPARED_BATCH_EXECUTION:
+                    component.setForeground(Color.BLUE);
+                    component.setText("PB");
+                    break;
+                case BASE_PREPARED_STMT:
+                    component.setForeground(Color.CYAN);
+                    component.setText("PS");
+                    break;
+                case PREPARED_QUERY_STMT:
+                    component.setForeground(Color.GREEN);
+                    component.setText("PQ");
+                    break;
+                case TRANSACTION:
+                    component.setForeground(Color.DARK_GRAY);
+                    component.setText("TX");
+                    break;
             }
 
             component.setToolTipText(value + " (use STATEMENTTYPE=" + statementType.getId()
                     + " in the \"Advanced filter\")");
+        } else if (LogRepositoryConstants.TRANSACTION_ISOLATION_COLUMN.equals(dataModel.getColumnName(columnModelIndex))) {
+            final int modelRowIndex = table.convertRowIndexToModel(row);
+            final Integer transactionIsolation = (Integer) dataModel.getValueAt(modelRowIndex, columnModelIndex);
+            if (transactionIsolation != null ) {
+                switch (transactionIsolation) {
+                    case 1:
+                        component.setText("RU");
+                        component.setToolTipText("Read Uncommitted");
```

Comment:
I suggest to add the constant number in parentheses so that if one wants to apply a filter the value to use will be easily visible.
e.g "Read Uncommitted (1)"